packages/unmock-core/src/service/spy/index.ts

```diff
@@ -0,0 +1,63 @@
+import { SinonSpy, spy as sinonSpy } from "sinon";
```

Comment:
This is a bit confusing (having both `sinonSpy` and `SinonSpy`). How about `import { SinonSpy as SinonSpyType, ...`?

Reply:
Renamed.